Dizzy/Entities/GoogleMap/GoogleMap.swift

```diff
@@ -32,10 +33,21 @@ final class GoogleMap: MapType {
     
     var googleMapView: GMSMapView
     
+    var showUserLocation: Bool = false {
+        didSet {
+            googleMapView.isMyLocationEnabled = showUserLocation
+        }
+    }
+    
     init() {
         let googleMapAPIKey = "AIzaSyCBhvRQXfqyNUQ_y9vm9Ikxi_t_U51ZaYI"
         GMSServices.provideAPIKey(googleMapAPIKey)
         googleMapView = GMSMapView()
+        do {
```

Comment:
please insert the `do` in to the if statement

Reply:
Done